bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/api/DatahubTariffFilter.java

```diff
@@ -18,7 +18,8 @@
 import org.eclipse.jdt.annotation.NonNullByDefault;
 
 /**
- * Filter for the DatahubPricelist dataset.
+ * Filter for the {@link org.openhab.binding.energidataservice.internal.api.dto.DatahubPricelistRecords}
```

Comment:
This was already addressed in #15651 where the link was removed. The name of the dataset is DatahubPricelist while the name of the corresponding DTO is DatahubPricelistRecords. I would therefore prefer to have this change reverted, and probably you need to rebase as well.

Reply:
I rebased and got it wrong.... I will revert.